.github/workflows/cargo-doc.yaml

```diff
@@ -6,6 +6,11 @@ on:
       - master
   pull_request: {}
 
+# Cancel previous runs of this workflow when a new commit is added to the PR, branch or tag
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
```

Comment:
So if not triggered on a `pull_request`, the expression `github.event.pull_request.number || github.ref` should fall back to the ref name and not just crash, right?

Reply:
Yes exactly